### ISSUE TITLE
Tab vs Space indentation

### ISSUE THREAD

**Issue by Drup** [darioteixeira/lambdoc#10]
I know it may be an annoying remark, but, can the codebase switch to spaces please ? 

Most OCaml codebases I have encountered are using spaces and now that ocp-indent is around, you can customize your indention on a project basis by putting a `.ocp-indent` at the root of your project. 

Editing your tab-indented code is very annoying for several reasons:
 - You use tab for indentation (that's fine) *and* alignment, which leads to the next point
 - According to the alignment in the code, you display 8 spaces for a tab, and let's just say it's extremely inconfortable for my screen size (and extremely inconfortable, period).

I'm not a fervent defender of spaces for indentation (as opposed to alignment) but in the case of OCaml, considering the ecosystem and the tooling, I don't see any reasons to stick to tabs.

**Comment by darioteixeira** [comment]
Hey @drup, I've just committed [7d51df9](https://github.com/darioteixeira/lambdoc/commit/7d51df95fcee508c6e1f8bbcbe2b3263090164cf), which should have taken care of this issue.

I've opted for 4-space indentation.  2 spaces seems more common in the OCaml community, but I find it too small to gauge nesting comfortably.  In any case, the more important thing is that the TAB character has been replaced with actual spaces.
